mos-platform/geos-cbm/geos_constants.h

```diff
@@ -0,0 +1,757 @@
+/***********************************************************************
+ * geos_constants.h
+ *
+ * This file contains #defines for use in GEOS applications
+ * with llvm-mos C compiler.
+ *
+ * Transcribed from geoProgrammer manual by DiSc.
+ *
+ ***********************************************************************/
+
+#ifndef GEOS_CONSTANTS_H
+#define GEOS_CONSTANTS_H
+
+/***********************************************************************
+ * Miscellaneous defines
+ ***********************************************************************/
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#if 0
```

Comment:
This section is not needed.

Reply:
I am not sure what type uint8_t now is, when including stdint.h. One nice effect when defining it myself as unsigned char was that one could give a char* instead of an uint8_t*, therefore saving casts in the source code...

For now i will remove the #if 0 block, but maybe later I need to refine uint8_t to unsigned char again.

Reply:
`uint8_t` and `unsigned char` can be used interchangeably: https://godbolt.org/z/MEa3cezGY